game_eggs/steamcmd_servers/v_rising/egg-v-rising.json

```diff
@@ -0,0 +1,231 @@
+{
+    "_comment": "Pterodactyl V Rising Egg ~ David Wolfe (Red-Thirten) and Kapatheus ~ 2022-05-26",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "name": "V Rising",
+    "author": "rehlmgaming@gmail.com",
+    "description": "Awaken as a vampire. Hunt for blood in nearby settlements to regain your strength and evade the scorching sun to survive. Raise your castle and thrive in an ever-changing open world full of mystery. Gain allies online and conquer the land of the living.",
+    "features": [
+        "steam_disk_space"
+    ],
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:wine_latest"
+    ],
+    "file_denylist": [],
+    "startup": "wine64 .\/VRisingServer.exe -persistentDataPath save-data -address 0.0.0.0",
```

Comment:
you dont need wine64, but you should start it with xvfb-run -a. Also you can overwrite a lot of variables

`xvfb-run -a wine ./VRisingServer.exe -serverName "{{SERVER_NAME}}"  -saveName "{{SAVE_NAME}}" -persistentDataPath ./server-data/ -maxConnectedUsers {{MAX_USERS}} -maxConnectedAdmins {{MAX_ADMINS}}`

Reply:
> you dont need wine64. you can use wine

Everything I read online implies 64-bit applications should be run with with `wine64`. Is there a benefit to using `wine`?

> xvfb-run -a

What does this do exactly, and why is it needed? The server seemingly runs fine without it.

> Also you can overwrite a lot of variables

Yes, but I found it ultimately misleading for the end user if they ever open up `ServerHostSettings.json` to take a look. The startup flags do not cause their values to be written to the `ServerHostSettings.json` file, and there are some settings in the file that cannot be set by flags. So, ultimately, I felt it was cleaner to have every setting consolidated to the file instead of disjointed.

Reply:
you also should add a dir in save-data called Settings

p.e.
`mkdir -p $HOME/server-data/Settings`

with `persistentDataPath` the server searches for the config in `$HOME/server-data/Settings`

take care about: https://github.com/StunlockStudios/vrising-dedicated-server-instructions/issues/26

Reply:
> you also should add a dir in save-data called Settings
> take care about: https://github.com/StunlockStudios/vrising-dedicated-server-instructions/issues/26

Please stop leaving incorrect comments that clearly indicate you have not read the egg in the PR. The egg's installation script already creates this directory correctly. The issue linked was user error of them putting the `adminlist.txt` file within the `server-data` directory when it should have been placed in the `server-data/Settings` directory (which this egg does correctly.

Reply:
admin.txt does not  work in  persistentDataPath. Thats what i want to say ;)

Reply:
My apologies. You are correct in that `adminlist.txt` and `banlist.txt` are not currently recognized from within the `~/save-data/` or `~/save-data/Settings/` directories. Commit [bd77d26](https://github.com/parkervcp/eggs/pull/1680/commits/bd77d26939a8b18887771d0d52fb0d4164996c93) should resolve this issue by removing them from the persistent data path and adjusting the README to point users to the correct path that is currently working.